src/libs/Navigation/Navigation.js

```diff
@@ -48,37 +49,52 @@ function setDidTapNotification() {
  * @returns {Boolean}
  */
 function canNavigate(methodName, params = {}) {
```

Comment:
Since `canNavigate` is no longer using `InteractionManager`, I don't see any reason why it would need to be asynchronous/return a promise. So let's roll that back and make it so that `canNavigate` is synchronous again

Reply:
Sure, that make sense to me also. Reverted back canNavigate to be synchronous again.